apps/src/p5lab/poetry/PoetryLibrary.js

```diff
@@ -321,7 +321,8 @@ export default class PoetryLibrary extends CoreLibrary {
       desiredSize,
       (desiredSize * (PLAYSPACE_SIZE - OUTER_MARGIN)) / fullWidth
     );
-    const maxLineHeight = 30 / lines.length;
+    const maxLineHeight =
```

Comment:
The math/logic here is a bit hard to parse. Is it possible to add a comment or break it out a bit more maybe with variables or parenthesis?

Reply:
Great point.  I'll add a detailed comment.  (I did have some more parentheses but JsPrettier removed them :)